# Patrol service request
geometry_msgs/PoseStamped[] waypoints
float32 patrol_speed
int32 patrol_cycles
bool return_to_start
float32 timeout_seconds
---
# Patrol service response
bool success
string message
float32 execution_time
int32 completed_cycles
geometry_msgs/PoseStamped final_pose
string task_id
